Sources/ApolloCodegenLib/ASTCondition.swift

```diff
@@ -1,18 +0,0 @@
-import Foundation
```

Comment:
@designatednerd Are these AST objects being used by the Javascript Frontend? Is there even anything that is creating them currently, other than unit tests for the experimental codegen? I think we can delete these from `main`, but just want to make sure.

Reply:
They should not be, this was being created off reading everything in using the TS CLI, you should be able to nuke them from orbit